src/main/resources/alfresco/mimetype/mimetype-map.xml

```diff
@@ -22,6 +22,7 @@
 
          <mimetype mimetype="application/vnd.ms-project" display="Microsoft Project">
             <extension default="true">mpp</extension>
+            <extension default="true">mpt</extension>
```

Comment:
`mpp` should remain the default, I think.

Reply:
`mpt` is not default anymore.